arch/arm/src/armv7-a/arm_syscall.c

```diff
@@ -238,7 +238,9 @@ uint32_t *arm_syscall(uint32_t *regs)
            * set will determine the restored context.
            */
 
+#if defined(CONFIG_ARCH_FPU)
```

Comment:
don't need since arm_restorefpu become the empty macro if CONFIG_ARCH_FPU isn't defined